__tests__/topics.test.js

```diff
@@ -327,3 +327,65 @@ describe('CORE: POST /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+describe('CORE: PATCH /api/articles/:article_id', () => {
+  test('201: should respond with object', () => {
+    const newVote = 10;
+    const patchInput = { inc_votes: newVote };
+    return request(app)
+      .patch('/api/articles/9')
+      .send(patchInput)
+      .expect(201)
+      .then(({ body }) => {
+        expect(typeof body.updatedArticle).toBe('object');
+      });
+  });
+  test('201: when passed a value of 10 should increment votes by ten', () => {
+    const newVote = 10;
+    const patchInput = { inc_votes: newVote };
+
+    return request(app)
+      .patch('/api/articles/9')
+      .send(patchInput)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.updatedArticle.votes).toEqual(10);
+      });
+  });
+  test('201: when passed a value of -10 should decrement votes by ten', () => {
+    const newVote = -10;
+    const patchInput = { inc_votes: newVote };
+
+    return request(app)
+      .patch('/api/articles/9')
+      .send(patchInput)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.updatedArticle.votes).toEqual(-10);
+      });
+  });
+  test('404: responds with error when passed an article_id that is valid type but non-existent', () => {
+    const newVote = 10;
+    const patchInput = { inc_votes: newVote };
+
+    return request(app)
+      .patch('/api/articles/999')
+      .send(patchInput)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not found');
+      });
+  });
+  test('400: responds with error when passed an article_id that is an invalid type', () => {
+    const newVote = 10;
+    const patchInput = { inc_votes: newVote };
+
+    return request(app)
+      .patch('/api/articles/egg')
+      .send(patchInput)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad request');
+      });
+  });
+});
```

Comment:
Can you add another test/s which checks the errors are being handled when the user provides an incorrect body object `{ inc_votes: newVote }` (for example no body is provided or the votes property is the wrong datatype)